docs/basics.rst

```diff
@@ -99,7 +99,7 @@ Here's a schema that will help us meet those requirements:
        
 The schemas used by Deform come from a package named :term:`Colander`.  The
 canonical documentation for Colander exists at
-http://docs.pylonsproject.org/projects/colander/dev/ .  To compose complex
+http://docs.pylonsproject.org/projects/colander/ .  To compose complex
```

Comment:
The correct link is:
https://docs.pylonsproject.org/projects/colander/en/latest/